src/LaminasViewRenderer.php

```diff
@@ -4,21 +4,15 @@
 
```

Comment:
I skimmed the changes on this file, but I don't have a mental map of all the BC implications of these changes, so I can't confidently give a LGTM on it